deploy/helm_charts/envs/datagemma.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DNS
+ip:
+domain: datagemma.datacommons.org
+
+# Helm config
+project: "datcom-datagemma"
+cluster_prefix: "website"
+
+namespace:
+  name: "website"
+website:
+  flaskEnv: custom
+  replicas: 20
+  redis:
+    enabled: true
+    configFile: |
+      {
+        "us-central1": {
+          "host": "10.98.18.227",
+          "port": "6379"
+        }
+      }
+  nodePool: "default-pool"
+serviceAccount:
+  name: website-ksa
+ingress:
+  enabled: true
+  internal: true
+nl:
+  enabled: true
+  memory: "10G"
+  workers: 3
+  env:
+    default_indexes:
+      - base_uae_mem
+    enabled_indexes:
+      - medium_ft
+      - sdg_ft
+      - undata_ft
+      - base_uae_mem
+    vertex_ai_models:
+      uae-large-v1-model:
+        project_id: datcom-nl
+        location: us-central1
+        prediction_endpoint_id: "1309973546495115264"
```

Comment:
can we actually use this endpoint_id instead: 4332900447158272000

This is a datagemma specific endpoint: https://screenshot.googleplex.com/cyZ5XSrs6kBf7pW

Reply:
done